iOS/docs/kickoff/configure-ci/README.md

```diff
@@ -0,0 +1,171 @@
+CI configuration for iOS projects
+-----------------
+
+This is a tutorial explaining how to configure continuous integration for iOS projects from scratch. Some steps can't be followed without being granted access to certain services: request them to an iOS' technical leader:
+
+* [Pablo Giorgi](mailto:pablo.giorgi@wolox.com.ar)
+* [Guido Marucci Blas](mailto:guidomb@wolox.com.ar)
+
+The following steps will guide you to setup a new project in [Bitrise](https://bitrise.io). Note that this guide is intended for standard projects, so if any custom configuration is needed some steps may change.
+
+## Check repo files...
+
+### Check the repository’s `bitrise.yml`. 
+
+It can be found [here](https://github.com/Wolox/ios-base-project/blob/master/bitrise.yml) in the `ios-base-project` root directory.
+
+Replace `REPO_SLUG`, `BITRISE_PROJECT_PATH` and `BITRISE_SCHEME` values with your app’s values.
+Keep in mind that the bitrise steps versions may be outdated, so check the [last versions](https://github.com/bitrise-io/bitrise-steplib/tree/master/steps) inside each steps folder and update the file accordingly. Example: if `bitrise.yml` contains a step `script@1.1.3`, check in that folder, in the step `script` if `1.1.3` is the last version.
```

Comment:
Keep in mind that the bitrise steps versions may be outdated, so check the [latest versions](https://github.com/bitrise-io/bitrise-steplib/tree/master/steps) inside each steps folder and update the file accordingly. Example: if `bitrise.yml` contains a step `script@1.1.3`, check in that folder, in the step `script` if `1.1.3` is the latest version.

Reasoning: 
- Last refers to the end of something, the tail. Latest refers to something that can be updated.